CONTRIBUTING.md

```diff
@@ -59,6 +59,12 @@ To install `tox`, run:
 pip install tox
 ```
 
+You can also run tox with `uv` support. By default [tox.ini](./tox.ini) will automatically create a provisioned tox environment with `tox-uv`, but you can install it at host level:
+
+```sh
+pip install tox-uv
```

Comment:
mind adding an example of using `uv` to run tests in a single package, or is this just plumbing under tox, so no uv based entrypoints..

Reply:
This is plumbing for tox

Reply:
thx